### DISCUSSION TITLE
How to change parameters.yml from controller?

### DISCUSSION THREAD

**Discussion by GeorgNation** [symfony/symfony#53161] [Q&A] [answered]
Hello there. I want to make a website configuration page on my website, but I can't find the way to realize this feature. How do I make it? Thanks in advance.

**Comment by stof** [comment]
Given that changing the DIC configuration requires a cache clearing, it is a bad idea to do it in a controller IMO (even more if your application has several servers behind a load balancer as you will have to replicate the update *and* the cache clearing on all servers).

Thus making this file invalid can make the whole application unusable by making it impossible to boot the kernel. So this is quite risky.

**Reply by GeorgNation** [reply]
The essence of the question is that, how to change the data (they mean for example the name of the site, etc.) of the site from under the controller, so that it was possible in one page to change the information, and it would apply, other means except as parameters.yml I do not know.

**Reply by mbabker** [reply, marked as answer]
Does this information need to be stored in `parameters.yml` and used when the container is compiled?  If the answer's no, then you're better off saving that config info somewhere else (i.e. the database) and pulling from there when you need it.
